CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Next release
+* Add 'Edit' button to objects in a sortedM2M field
+* Add '?preview=1' to sitemap on-site URLs to allow administrators to view offline pages.
```

Comment:
Can you change the destination branch for this PR from `master` to `develop`? This confuses the diff somewhat.

Reply:
Yeah, I assumed it would try to merge into the branch you split off from